dts/bindings/tcpc/st,stm32-ucpd.yaml

```diff
@@ -0,0 +1,71 @@
+# Copyright 2021 The Chromium OS Authors
+# SPDX-License-Identifier: Apache-2.0
+
+description: ST STM32 family USB Type-C / Power Delivery.
```

Comment:
nit: `description: |` needs to be used for multiline comments, ref https://docs.zephyrproject.org/latest/guides/dts/bindings.html#rules-for-mainline-bindings

Reply:
done